_data/simple-icons.json

```diff
@@ -3450,7 +3451,7 @@
         {
             "title": "Gmail",
             "hex": "EA4335",
-            "source": "https://fonts.gstatic.com/s/i/productlogos/gmail_2020q4/v8/192px.svg"
+            "source": "https://about.google/brand-resource-center/logos-list/"
```

Comment:
I'm not sure this is a good source replacement since the logo is in PNG in this page.

Reply:
- It is never as easy to create a quality icon from a PNG than from a SVG
- As stated in the guidelines, the source is supposed to represent the image used to create the icon. Since it was created from a SVG, linking to a PNG is not representative of the icon added

Since we disagree on this point, we might need another @simple-icons/maintainers opinion

Reply:
I definitely agree that in general an official SVG is preferred over any raster file (unless, e.g., the only available official SVG is clearly malformed).

That said, the current source URL isn't ideal either.  @jorgeamadosoria, did you find any indication of a webpage on which the current source can be found?

Reply:
> That said, the current source URL isn't ideal either. @jorgeamadosoria, did you find any indication of a webpage on which the current source can be found?

I agree, but changing a not so good source for another not so good source is not ideal. For icons where we're not able to find better source, we might open new issues for them. At least, we'll be able to merge this PR for icons where the new source makes sense.

Reply:
> I definitely agree that in general an official SVG is preferred over any raster file (unless, e.g., the only available official SVG is clearly malformed).
> 
> That said, the current source URL isn't ideal either. @jorgeamadosoria, did you find any indication of a webpage on which the current source can be found?

No. the fonts.gstatic does not seem to be a host that offers direct downloads. 
In the cases I've found pages, I have changed it to them (see TensorFlow or Jamboard, for example).
In general I'm perfectly fine with the PNGs to substitute these particular icons. The ones there are were created from SVGs, but there are no official SVGs for this.

It may be possible to find SVGs for Google icons in places like seeklogo.com or vectorlogos.com, but the question then becomes: official PNG > unofficial SVG, or viceversa?

Reply:
> In general I'm perfectly fine with the PNGs to substitute these particular icons.

The source is used to indicate where we _sourced_ the icon. Linking to PNG version seems wrong to me when SVG files have been used. The current source is far from ideal, but I would not update the source if we cannot find something better.

Reply:
In the interest of getting things that can be merged, merged as quickly as possible, I would indeed suggest to separate the cases where the new source only has vector files into a separate PR. What do you think about that @jorgeamadosoria?

> The ones there are were created from SVGs, but there are no official SVGs for this.

I'm confused by this statement. How do you know that?

Reply:
I don't know if there are, I was not able to find official SVGs.

Reply:
going to change this one for Commons Wiki sources.
Same with the other Gmail icons.

Reply:
if that's fine, please resolve the corresponding commments and also mark them in the list to keep track of what's been checked.

Reply:
Changing an official source from the main company for a less official source might be an option, but I'm not sure I would do this in this PR.

As mentioned before, I think it would be a good idea to keep those as-is for now so we can merge this PR soon. You could open a new issue for brands that need discussion.

Reply:
The  Commons Wiki source is an SVG, they are the same as the ones from where the. current icons are derived, and they are (relatively) hi-res. 
They are also possible to reference from a page, which sources like gstatic are not.